modules/python/Makefile.style

```diff
@@ -1,5 +1,5 @@
 BLACK=$(call which,black)
-ISORT=$(call which,isort)
+ISORT=$(call which,isort) --multi-line=3 --trailing-comma --force-grid-wrap=0 --use-parentheses --line-width=88
```

Comment:
My expectation was that you should have all this in your setup.cfg or .isort.cfg